index.html

```diff
@@ -24,16 +24,32 @@
         <header class="header">
             <div class="logo"><a href="#">MD SAZZAD</a></div>
             <nav class="nav-icon">
-                <a href="navbar.html" target="_self"><img src="img/bars/Union.png" alt=""></a>
+                <a class="hambarg-icon" href="navbar.html" target="_self"><img src="img/bars/Union.png" alt=""></a>
                 
+                <ul class="navbar-ul">
+                    <li><a href="#">Portoflio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li>
+                        <a href="#">
+                            <img src="icon/Mail.png" alt="">
+                        </a>
+                    </li>
+                </ul>
             </nav>
+           
         </header>
-    <!--hero section-->
-    <section class="hero">
-        <h1 class="title">Hey There.<br>I'm Sazzad</h1>
+    <!--main section-->
+    <section class="main">
+        <div class="main-container">
+            <div class="main-text">
+                <h1 class="title">Hey There.<br>I'm Sazzad</h1>
         <h2 class="sub-title">I am a Softwaer Developer</h2>
         <p class="detail">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
+            </div>
+            <div class="main-img"></div>
```

Comment:
- [ ] Kindly add this background image in CSS instead of HTML since it's not part of the content 👍